yearn/apy/curve/simple.py

```diff
@@ -81,12 +83,35 @@ def simple(vault, samples: ApySamples) -> Apy:
     if vault.vault.address == "0x46AFc2dfBd1ea0c0760CAD8262A5838e803A37e5":
         boost = 1
 
-
+    # TO-DO: come up with cleaner way to deal with these new gauge rewards
```

Comment:
it's normally written just `TODO`, which doesn't make sense but is just the convention..

Reply:
Done